solutions/dedicated/README.md

```diff
@@ -0,0 +1,64 @@
+# IBM Cloudant Dedicated on IBM Cloud Dedicated
+
+Solutions for an IBM Cloudant dedicated instance which supports provisioning the following resources:
+
+- A resource group, if one is not passed in.
+- An IBM Cloudant Dedicated instance in IBM Cloud Dedicated
+- A Cloudant database
+
+## Before you begin
+
+* You need an [IBM Cloudant Dedicated](https://cloud.ibm.com/docs/Cloudant?topic=Cloudant-ibm-cloud-dedicated) Hardware plan instance in order to provision the Cloudant instance in.
+
+<!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
```

Comment:
This infor will appear with the DA in the cloud catalog, so we want to make sure that it makes sense in that context.

Reply:
Removing the pre-commit-terraform docs hook, we don't want to display this information (Same as in SLZ module).